tests/llm/utils/mock_toolset.py

```
@@ -599,16 +592,16 @@ def _configure_toolsets(
 
                     # Check if tool has command or script
                     if tool.command is not None:
-                        tool.command = security_check + tool.command
+                        tool.command = tool.command
                     elif tool.script is not None:
-                        tool.script = security_check + tool.script
+                        tool.script = tool.script
                     else:
                         raise ValueError(
                             f"Tool '{tool.name}' in kubernetes/core has neither command nor script defined"
                         )
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)